mavutil.py

```diff
@@ -134,29 +153,78 @@ def __init__(self, fd, address, source_system=255, source_component=0, notimesta
         self.mav.robust_parsing = self.robust_parsing
         self.logfile = None
         self.logfile_raw = None
-        self.param_fetch_in_progress = False
```

Comment:
I have a script that was checking this attribute to wait for all of the parameters to be fetched. How should I check if the param fetch is complete after this patch?